### PR TITLE
Added drush compatibility

### PR DESCRIPTION
**Pull request by lucsrods**
Aliases and function for drush to help the drupal workflow.

### CODE REVIEW

File: plugins/drush/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Drush plugin #
```
Comment (RobLoach) [inline plugins/drush/README.md]: Don't need the suffixing #

**Comment by RobLoach** [comment]
Could likely bring in the autocomplete stuff in https://github.com/drush-ops/drush/blob/8.0.1/drush.complete.sh .

**Comment by lucasar** [comment]
@RobLoach hey man, thanks for the updates, and sorry for the delay. Regarding your suggestion i'll take a look and see what i can integrate in both scripts.

File: plugins/drush/drush.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Drush support.
+
+function dren() {
+	drush en $@ -y
```
Comment (mcornella) [inline plugins/drush/drush.plugin.zsh]: This doesn't have the same indentation

File: plugins/drush/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Description ##
+This plugin offers aliases and functions to make the work with drush easier and more productive.
+
+## How to use ##
```
Comment (mcornella) [inline plugins/drush/README.md]: This whole section should be restructured to use markdown tables. [See the `xcode` plugin for an example](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/xcode/README.md#aliases).

**Comment by revagomes** [comment]
:+1:

**Comment by lucasar** [comment]
Hey there @revagomes, thanks for the interest and suggestion, I'll take a look on their script and see what we can be using.

**Pull request merged**